src/Workspaces/Core/Portable/Shared/TestHooks/AsynchronousOperationListener.cs

```diff
@@ -40,28 +40,39 @@ public AsynchronousOperationListener(string featureName, bool enableDiagnosticTo
             TrackActiveTokens = Debugger.IsAttached || enableDiagnosticTokens;
         }
 
-        public async Task<bool> Delay(TimeSpan delay, CancellationToken cancellationToken)
+        public Task<bool> Delay(TimeSpan delay, CancellationToken cancellationToken)
```

Comment:
💡  This implementation is only used in testing. Is there any downside to changing it (e.g. for debugging scenarios)?

Reply:
If it's only used in testing, I'll put it back to the way it was for simplicity (I wasn't sure, and so figured I'd tweak it while I was here).  The new version should be identical in behavior to the old, but if it's just for testing, there's little to no gain for the increased complexity.